src/AltinnCore/Designer/Controllers/HomeController.cs

```diff
@@ -68,7 +68,7 @@ public ActionResult StartPage()
         }
 
         /// <summary>
-        /// The default action presenting a list of available services when the user is logged in
+        /// The default action presenting a list of available applications when the user is logged in
```

Comment:
you are mixing the use of 'app' and 'application' in comments. Consistency would be nice but is not critical.